src/tests/build.proj

```diff
@@ -331,24 +338,107 @@
+
+      <MonoAOTCompilerDefaultAotArguments Include="nimt-trampolines=2000" />
+      <MonoAOTCompilerDefaultAotArguments Include="ntrampolines=40000" />
+      <MonoAOTCompilerDefaultAotArguments Include="nrgctx-fetch-trampolines=256" />
+      <MonoAOTCompilerDefaultAotArguments Include="ngsharedvt-trampolines=4400" />
+      <MonoAOTCompilerDefaultAotArguments Include="nftnptr-arg-trampolines=4000" />
+      <MonoAOTCompilerDefaultAotArguments Include="nrgctx-trampolines=40000" />
+
+      <MonoAOTCompilerDefaultProcessArguments Include="-O=gsharedvt" />
+    </ItemGroup>
+
+    <PropertyGroup>
+      <_AOTMode Condition="'$(MonoForceInterpreter)' != 'true'">Full</_AOTMode>
+      <_AOTMode Condition="'$(MonoForceInterpreter)' == 'true'">JustInterp</_AOTMode>
+      <AotArguments>@(MonoAOTCompilerDefaultAotArguments, ';')</AotArguments>
+      <ProcessArguments>@(MonoAOTCompilerDefaultProcessArguments, ';')</ProcessArguments>
+      <_MobileIntermediateOutputPath>$([MSBuild]::NormalizeDirectory($(IntermediateOutputPath), 'mobile'))</_MobileIntermediateOutputPath>
+    </PropertyGroup>
+
+    <Message Importance="High" Text="!!!naricc_debug!!!: Build dir: $(BuildDir)/testdir*/*.dll"/>
+
+    <ItemGroup>
+        <_AppleAssembliesInternal Include="$(BuildDir)/**/*.dll" />
+
+        <_AppleAssembliesInternal>
+            <_InternalForceInterpret Condition="'$(MonoForceInterpreter)' == 'true' and '%(FileName)%(Extension)' != 'System.Private.CoreLib.dll'">true</_InternalForceInterpret>
+            <_IsNative>false</_IsNative>
+        </_AppleAssembliesInternal>
+
+        <_AotInputAssemblies Include="@(_AppleAssembliesInternal)"
+                            Condition="'%(_AppleAssembliesInternal._InternalForceInterpret)' != 'true'">
+            <AotArguments>$(AotArguments)</AotArguments>
+            <ProcessArguments>$(ProcessArguments)</ProcessArguments>
+        </_AotInputAssemblies>
+
+        <_AOT_InternalForceInterpretAssemblies Include="@(_AppleAssembliesInternal->WithMetadataValue('_InternalForceInterpret', 'true'))" />
+        <_AppleAssembliesInternal Remove="@(_AppleAssembliesInternal)" />
+    </ItemGroup>
+
+    <MakeDir Directories="$(_MobileIntermediateOutputPath)" />
+
+    <PropertyGroup Condition="'$(RuntimeFlavor)' == 'Mono'">
+      <_MonoAotCrossCompilerPath>$([MSBuild]::NormalizePath($(MonoAotCrossDir), 'mono-aot-cross'))</_MonoAotCrossCompilerPath>
+      <_MonoAotCrossCompilerPath Condition="$([MSBuild]::IsOSPlatform('WINDOWS'))">$(_MonoAotCrossCompilerPath).exe</_MonoAotCrossCompilerPath>
+      <DevTeamProvisioning>-</DevTeamProvisioning>
+    </PropertyGroup>
+    <ItemGroup Condition="'$(RuntimeFlavor)' == 'Mono'">
+      <MonoAotCrossCompiler Include="$(_MonoAotCrossCompilerPath)" RuntimeIdentifier="$(TargetOS.ToLowerInvariant())-$(TargetArchitecture.ToLowerInvariant())" />
+    </ItemGroup>
+
+    <MonoAOTCompiler Condition="'$(RunAOTCompilation)' == 'true'"
```

Comment:
@mdh1418 Can `AppleApp.targets` be used here?

Reply:
@kotlarmilos It might be able to be used, I haven't attempted to integrate that in. It would definitely simplify this proj and not have redundant stuff.